cristin-project/src/test/resources/nvaApiGetResponseOneNvaProject.json

```diff
@@ -140,5 +140,23 @@
     "authorityName" : {
       "en" : "Regional committees for medical and health research ethics"
     }
-  } ]
+  } ],
+  "creator": {
+    "type": "ProjectCreator",
+    "identity": {
```

Comment:
But to be honest, isn't just Identifier better? what happens when email or affiliation changes?

Reply:
We want to include creator in list of participants, therefore the same model as them and the possibility to have both name of the person and name of the affiliated organization